README.md

```diff
@@ -34,6 +34,41 @@ to know more about what happens under the hood in **OpenWISP 2**.
 
 For more information see also [Applying the Unix Philosophy to Django projects: a report from the real world](https://www.slideshare.net/FedericoCapoano/applying-the-unix-philosophy-to-django-projects-a-report-from-the-real-world).
 
+System Requirements
+===================
+
+The following specifications will run a new, *empty* instance of OpenWISP.
+Please ensure you account for the amount of disk space your use case will require, e.g. allocate
+enough space for users to upload floor plan images.
+
+Hardware (Minimum, excluding uploaded data)
+-------------------------------------------
+- 1GB Memory
+- 300MB free disk space
+- Single core processor
+
+Hardware (Recommended)
+----------------------
+- 2GB Memory
+- Disk space - depends on the projected size of your database and uploaded photo images
+- Dual core processor
+
+Software
+--------
+Generally a fresh installation of one of the supported operating systems is sufficient; no pre-configuration required.  The
+Ansible Playbook will install and configure all dependencies and leave you with a running OpenWISP installation.
+
+Tips: Ensure the hostname of your device matches what is in your Ansible configuration file.  Also, please ensure
```

Comment:
I would use a different word instead of `device` to avoid confusion with devices as e.g. routers. What about `target machine`?

Reply:
good call